PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-appservice/src/main/java/com/microsoft/azure/toolkit/intellij/legacy/appservice/AppServiceComboBox.java

```diff
@@ -44,7 +44,7 @@
     public AppServiceComboBox(final Project project) {
         super(false);
         this.project = project;
-        this.setRenderer(new AppComboBoxRender());
+        this.setRenderer(new AppComboBoxRender(false));
```

Comment:
1. add comment to acr name instead of disable it
2. add message and action when user select acr without admin user